README.rst

```diff
@@ -1,44 +1,16 @@
-*fMRIPrep*: A Robust Preprocessing Pipeline for fMRI Data
+*fMRIPrep/rodents*: Analysis-grade BOLD fMRI data of rats
 =========================================================
-
-This pipeline is developed by the `Poldrack lab at Stanford University
-<https://poldracklab.stanford.edu/>`_ for use at the `Center for Reproducible
-Neuroscience (CRN) <http://reproducibility.stanford.edu/>`_, as well as for
-open-source software distribution.
-
-.. image:: https://img.shields.io/badge/docker-poldracklab/fmriprep-brightgreen.svg?logo=docker&style=flat
-  :target: https://hub.docker.com/r/poldracklab/fmriprep/tags/
-  :alt: Docker image available!
-
-.. image:: https://codeocean.com/codeocean-assets/badge/open-in-code-ocean.svg
-  :target: https://doi.org/10.24433/CO.ed5ddfef-76a3-4996-b298-e3200f69141b
-  :alt: Available in CodeOcean!
-
-.. image:: https://circleci.com/gh/poldracklab/fmriprep/tree/master.svg?style=shield
-  :target: https://circleci.com/gh/poldracklab/fmriprep/tree/master
-
-.. image:: https://readthedocs.org/projects/fmriprep/badge/?version=latest
-  :target: http://fmriprep.readthedocs.io/en/latest/?badge=latest
-  :alt: Documentation Status
-
-.. image:: https://img.shields.io/pypi/v/fmriprep.svg
-  :target: https://pypi.python.org/pypi/fmriprep/
-  :alt: Latest Version
-
-.. image:: https://img.shields.io/badge/doi-10.1038%2Fs41592--018--0235--4-blue.svg
-  :target: https://doi.org/10.1038/s41592-018-0235-4
-  :alt: Published in Nature Methods
-
-.. image:: https://img.shields.io/badge/RRID-SCR__016216-blue.svg
-  :target: https://doi.org/10.1038/s41592-018-0235-4
-  :alt: RRID:SCR_016216
+This tool adapts *fMRIPrep* to the use-case of rodent preclinical imaging -- 
+because MRI is some times done on species other than the *homo sapiens*.
 
 About
 -----
+Development was initiated by (soon-to-be) Dr. E. MacNicol in a research visit to the 
```

Comment:
This is the first time I've seen this, and it is very surreal ☺️

Reply:
We'll remove the surrealism (and those parenthesis) pretty soon :)